test/lib_management/refcount.jl

```diff
@@ -1,9 +1,9 @@
 @testset "Test Refcounting logic" begin
     @testset "basic refcount" begin
-        abs_so = abspath("nlp.so")
-        abs_json = abspath("nlp.json")
-        nlp = CasADiNLPModel(abs_so, "nlp.json")
-        nlp2 = CasADiNLPModel(abs_so, "nlp.json")
+        abs_so = abspath(joinpath(@__DIR__, "..", "nlp.so"))
+        abs_json = abspath(joinpath(@__DIR__, "..", "nlp.json"))
+        nlp = CasADiNLPModel(abs_so, abs_json)
```

Comment:
In principle it may actually be good to pass a relative path here for one of the NLPs and the absolute path for the other. The refcounting machinery uses absolute paths to keep track of duplicate loads. I do feel like this is a bit of an artificial test but I don't have any better ways to make sure that all the refcounting calls work as expected.